packaging/bundle-validation/validate.sh

```diff
@@ -185,6 +186,42 @@ test_kafka_connect_bundle() {
     kill $ZOOKEEPER_PID $KAFKA_SERVER_PID $SCHEMA_REG_PID
 }
 
+##
+# Function to test the hudi metaserver bundles.
+#
+# env vars (defined in container):
+#   SPARK_HOME: path to the spark directory
+##
+test_hudi_metaserver_bundles () {
+    echo "::warning::validate.sh setting up hudi metaserver bundles validation"
+
+    echo "::warning::validate.sh Start hudi metaserver"
+    java -jar $JARS_DIR/metaserver.jar & local METASEVER=$!
+
+    echo "::warning::validate.sh Start hive server"
+    $DERBY_HOME/bin/startNetworkServer -h 0.0.0.0 &
+    local DERBY_PID=$!
+    $HIVE_HOME/bin/hiveserver2 --hiveconf hive.aux.jars.path=$JARS_DIR/hadoop-mr.jar &
+    local HIVE_PID=$!
```

Comment:
we don't need hive server here as we don't need to run hiveql to validate

Reply:
cause the hive-site.xml is in, so if not start hive server, spark cannot start.

Reply:
cause the hive-site.xml is in, so if not start hive server, spark cannot start.

Reply:
ok we can fix this later to decouple hive from this testcase's setup

Reply:
+1. // todo:  decouple hive-site.xml from spark-deploy's conf dir

Reply:
https://issues.apache.org/jira/browse/HUDI-5621